src/cases/components/allocate-role/allocate-role.component.scss

```diff
@@ -0,0 +1,31 @@
+
+.govuk-back-link {
+  border-bottom: 1px solid #0b0c0c;
+  text-decoration: none;
+  &::before {
+    content: ' \25C0';
+    top: -10px;
+    left: 0px;
+    font-size: 12px;
+    border-color: transparent;
+    position: absolute;
+    transform: rotate(1deg);
+  }
+}
+
+.govuk-fieldset__heading {
```

Comment:
And again?? These are part of the global styles they do not need to be redefined

Reply:
See above. The header was appearing as normal text with no attributes